README.md

```diff
@@ -32,26 +44,68 @@ python run-demo.py
 
 ## Manual
 
-If you prefer to run the demo manually, set up a directory structure as follows:
+If you prefer to run the demo manually, follow the steps outlined below.
+You will set up a directory structure as follows:
 
 ```
 .
 ├── keys
+├── gpg-dir
 └── repo
 ```
 
-`keys` must be copied from this repository. You must also create two
+Where `keys` will be copied from this repository. You will create two
 Git-compatible signing keys, one that is authorized for the demo policy and one
-that is not. Finally, set your working directory to `repo`.
+that is not. You will see how policies are created, commit changes and run
+`gittuf` to verify, both on the happy and failing path:
 
 ```bash
+# Temporary playground
+cd $(mktemp -d)
+
+mkdir {keys,repo,gpg-dir}
+chmod 700 gpg-dir
+cp -r ${OLDPWD}/keys .
+
+cd repo
+export GNUPGHOME=../gpg-dir  # optional: set GPG home if you're using GPG keys from a non-default keyring
+
+# For more options see:
+# https://www.gnupg.org/documentation/manuals/gnupg-devel/Unattended-GPG-key-generation.html
+gpg --batch --gen-key <<EOF
```

Comment:
we wanted to avoid cluttering up the default keyring with these keys. Maybe we should more explicitly call out that they may want to use a test keyring here?

Reply:
Or that if they already have two keys, they don't need to create new ones at all.

Reply:
And you're right below, in the manual mode they can also opt for two sigstore identities etc. Or one GPG key and one sigstore identity, whatever they prefer. It's only in the scripted run we want to fully control the key generation and usage process.

Reply:
Line 71 set's a new `GNUPGHOME` so the script as is will never pollute your actual key-ring.

Maybe I could add some explanation that users are welcome to skip the key generation and instead use their existing keys?

Reply:
That'd work well!